components/kserve/kserve.go

```diff
@@ -187,6 +182,12 @@ func (k *Kserve) configureServerless(instance *dsciv1.DSCInitializationSpec) err
 			return fmt.Errorf("ServiceMesh is need to set to 'Managed' in DSCI CR, it is required by KServe serving field")
```

Comment:
It is not related to this PR but L181, we should remove `Unmanaged` ? because serverless can work with existing Servicemesh (user's one)

Reply:
Can or should? @danielezonca

Reply:
What I remember is that the logic for `Unmanaged` is still _TODO_.

We cannot remove `Unmanaged`. The docs already guide users to use it if they want to use their own stack.